pkgs/tools/X11/srandrd/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchzip
+, libX11
+, libXrandr
+, libXinerama
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "srandrd";
```

Comment:
This variable is not used anywhere.

Reply:
I'm confused, are you claiming the pname attribute isn't needed?

Reply:
Sorry, I mean it can be used in url for example.